packages/react-components/src/molecules/__tests__/EventInfo.test.tsx

```diff
@@ -169,7 +169,7 @@ it('displays the team only once', () => {
     />,
   );
 
-  expect(screen.getAllByRole('listitem').length).toEqual(1);
+  expect(screen.getAllByText(/the team/i).length).toEqual(1);
```

Comment:
I'll move this test from here then, cause in the EventInfo component we have more <li> and the test keeps failing because the assertion always results in a number greater than 1

Reply:
The problem here is that I need to find a specific way to match this <li since there's other 2 <li in the page

Reply:
I could use ByRole and then use a .find, but it would be pretty much the same thing but with more code